client/src/pages/Dashboard/useStyles.ts

```diff
@@ -4,18 +4,119 @@ const drawerWidth = 240;
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    minHeight: '100vh',
-    '& .MuiInput-underline:before': {
-      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
-    },
+    // minHeight: '100vh',
+    // '& .MuiInput-underline:before': {
+    //   borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    // },
   },
+  // drawerWrapper: {
+  //   width: drawerWidth,
+  //   [theme.breakpoints.up('md')]: {
+  //     width: '300px',
+  //   },
+  // },
```

Comment:
We don't want to push up commented code to the main branch, if there is no need for these styles anymore then this can be removed and if you need to address this later you can refer to this branch to see the styling again.